{"pytest_version": "7.4.3", "$report_type": "SessionStart"}
{"nodeid": "tests42_23_3.py::test_list_of_tens", "location": ["tests42_23_3.py", 15, "test_list_of_tens"], "keywords": {"test_list_of_tens": 1, "tests42_23_3.py": 1, "Folder_23": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00018399200052954257, "start": 1700330384.8263652, "stop": 1700330384.8265498, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "5ac2135d602640b5bbca765f34c8de4d", "node": "<WorkerController gw1>"}
{"nodeid": "tests42_23_3.py::test_list_of_primes", "location": ["tests42_23_3.py", 4, "test_list_of_primes"], "keywords": {"test_list_of_primes": 1, "tests42_23_3.py": 1, "Folder_23": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00019241799600422382, "start": 1700330384.8263817, "stop": 1700330384.826575, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "5ac2135d602640b5bbca765f34c8de4d", "node": "<WorkerController gw0>"}
{"nodeid": "tests42_23_3.py::test_list_of_tens", "location": ["tests42_23_3.py", 15, "test_list_of_tens"], "keywords": {"test_list_of_tens": 1, "tests42_23_3.py": 1, "Folder_23": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/CodeLlama_13B/Q42/codellama_results_3/Folder_23/generated_answer.py", "lineno": 3, "message": "AttributeError: 'int' object has no attribute 'prime_factors'"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_tens():", "        list_of_tens = [10] * (69 + 100)", ">       assert prime_factors(list_of_tens) == {2,5}"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q42/codellama_results_3/Folder_23/tests42_23_3.py", "lineno": 18, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def prime_factors(numbers):", ">       return set(numbers[69].prime_factors())", "E       AttributeError: 'int' object has no attribute 'prime_factors'"], "reprfuncargs": {"args": [["numbers", "[10, 10, 10, 10, 10, 10, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q42/codellama_results_3/Folder_23/generated_answer.py", "lineno": 3, "message": "AttributeError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_tens():", "        list_of_tens = [10] * (69 + 100)", ">       assert prime_factors(list_of_tens) == {2,5}"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q42/codellama_results_3/Folder_23/tests42_23_3.py", "lineno": 18, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def prime_factors(numbers):", ">       return set(numbers[69].prime_factors())", "E       AttributeError: 'int' object has no attribute 'prime_factors'"], "reprfuncargs": {"args": [["numbers", "[10, 10, 10, 10, 10, 10, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q42/codellama_results_3/Folder_23/generated_answer.py", "lineno": 3, "message": "AttributeError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/CodeLlama_13B/Q42/codellama_results_3/Folder_23/generated_answer.py", "lineno": 3, "message": "AttributeError: 'int' object has no attribute 'prime_factors'"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.0001723189780022949, "start": 1700330384.8268352, "stop": 1700330384.8270082, "extras": [], "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "5ac2135d602640b5bbca765f34c8de4d", "node": "<WorkerController gw1>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
